department-of-veterans-affairs/caseflow
# Deepak/appeals 29211

Resolves [/APPEALS-29211](https://jira.devops.va.gov/browse/APPEALS-29211)

# Description
When a mail user selects the "Split package" option from the "Request package action" dropdown, a modal appeals and the user can select a reason for splitting

## Acceptance Criteria

## Testing Plan
1. Login as MAIL_TEAM_SUPERVISOR_ADMIN_USER or JOLLY_POSTMAN
2. Go to /queue/correspondence
3. Click on any veteran
4. Click on the "Request package action" from the dropdown
5. Click on "Split package from Caseflow"
6. Fill in the textbox or the other two responses
7. Click on the Confirm request button


# Frontend
## User Facing Changes
<img width="1587" alt="Screenshot 2023-12-12 at 2 27 50 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/132940479/e3818cf2-3a2e-4f29-9b2c-cbed72bfd52d">
<img width="1591" alt="Screenshot 2023-12-12 at 2 28 27 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/132940479/9f34c8b2-ca51-4213-9a88-a8a483a21626">
<img width="1608" alt="Screenshot 2023-12-12 at 2 31 23 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/132940479/731614bf-99ea-4b9e-8e76-8c3fa05f3a55">



## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
